### ISSUE TITLE
wiredepCopy cdoesn't copy to the right place

### ISSUE THREAD

**Issue by enricodeleo** [tlvince/grunt-wiredep-copy#1]
I tried this tool because grunt-bower-task doesn't work as good as it should be with grunt-wiredep. I thought this task would do the job but it has a huge problem. It doesn't actually create the new file in the target location but keep renaming the file itself.

This is the block I have in my Gruntfile.js

```
wiredepCopy: {
      target: {
        options: {
          dest: 'app',
          wiredep: {
            task: {
              src: [
                'www/**/*.html',
                'www/css/**/*.css'
              ],
              options: {
              }
            }
          }
        }
      },
    }
```

when I type `grunt wiredepCopy` it creates a new file in app/ but it crates a new file in the same directory of the source renaming it to whateverappjs. Basically it replaces the dot at the beginning of the extension with the src. 

Ideas? Any help would be much appreciated :)

**Comment by tlvince** [comment]
Haven't had time to diagnose, but on first glance, you're missing a `src` property on `wiredepCopy`'s `target`. Is this intended?

**Comment by enricodeleo** [comment]
Thank you for your reply, yes it is 'cause I had the issue above with '.' so I tried without but had no luck. As I said it seems like it search & replace for '.'.
ATM I resolved installing bower_components within the app folder (using ignore path in order to have wiredep working with a non default location). The drawback is an heavy directory but at least it works enough for dev purposes :)

**Comment by audunsol** [comment]
I have the same problem. @zi has resolved this issue, see: https://github.com/zi/grunt-wiredep-copy/blob/master/tasks/lib/wiredep-copy.js.

**Comment by tlvince** [comment]
Closed by e65ff64d7265809fa1a361acea3cff5b04630064
